puppet/modules/jenkins_job_builder/files/centos.org/pipelines/test/katello/nightly-upgrade.groovy

```diff
@@ -1,4 +1,3 @@
 def playBookVars() {
-    playBook = ['boxes': ['pipeline-upgrade-katello-nightly-centos7'], 'pipeline': 'upgrade_pipeline.yml', 'extraVars': ['pipeline_version': 'nightly', 'pipeline_os': 'centos7', 'pipeline_type': 'katello']]
-    return playBook
+    return upgradePipelineVars(type: 'katello', version: 'nightly', os: 'centos7')
```

Comment:
kinda feels like `os` should default to `centos7` :)

Reply:
I'd rather be explicit and add centos8 soon as well.